numpy/linalg/lapack_lite/clapack_scrub.py

```diff
@@ -1,4 +1,6 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python2.7
+from __future__ import with_statement
```

Comment:
This shouldn't be needed, since with statements are in 2.7

Reply:
Works now. I must have misinterpreted one of the error messages. I'll revert this change.